contracts/atomic-order-example/src/contract.rs

```diff
@@ -97,18 +100,18 @@ pub fn try_swap(
         false,
         true,
         &config.market_id,
```

Comment:
since we consume `subaccount_id` here why not consume `market_id` as well? And move both here instead of borrowing one and moving the other?